src/test/scala/com/microsoft/hyperspace/index/rules/JoinIndexRuleTest.scala

```diff
@@ -401,7 +401,12 @@ class JoinIndexRuleTest extends HyperspaceRuleTestSuite with SQLHelper {
       updatedPlan: LogicalPlan,
       indexPaths: Seq[Path]): Unit = {
     assert(treeStructureEquality(originalPlan, updatedPlan))
-    assert(basePaths(updatedPlan) == indexPaths)
+    val bP = basePaths(updatedPlan)
```

Comment:
Can you rename `basePaths` to `getRootPaths` and `val rootPaths = getRootPaths(udpatedPlan)`? (I had to look up what `basePaths` does. :)

Reply:
thanks @imback82 , done